en/manual/platforms/linux/setup-and-requirements.md

```diff
@@ -87,11 +87,11 @@ sudo pacman -S sdl2
 
 ---
 
-## .NET Core
+## .NET
 
-For information about how to install .NET Core, see the [.NET Core instructions for Linux](https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites).
+For information about how to install .NET, see the [.NET instructions for Linux](https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites).
 
-Make sure version 2.1.300+ and runtime 2.1+ is installed. To check which version you have installed, type:
+We recommend to install .NET 8. To check which version you have installed, type:
```

Comment:
Shall we rather add a link to https://stride-doc-staging.azurewebsites.net/latest/en/manual/requirements/index.html (I am adding the one from staging just in case there are more updates) so we update .NET version from one location? Including we can make that requirements page better?

Reply:
I think that is excellent point 😃. I wonder then if ".NET Core" header is longer needed  and the link to the requirements can be placed at the beginning of the article somehow

Reply:
Good point. Also, .NET Core is no more as it is just .NET, and yeah, it is a primary requirement, so it could be at the beginning. 

Also there is part `either Mono or .NET Core (it's OK to install both)`, do we need Mono?

Reply:
NET Runtime requirements should be equal across all platforms, and Mono from what I remember was NET Framework implementation so no 😃